seichi-onp-network-iap/services/docker-compose.yml

```
@@ -0,0 +1,14 @@
+networks:
+  cloudflared-internet:
+
+volumes:
+  cloudflared-home:
+
+services:
+  cloudflared:
+    image: ghcr.io/giganticminecraft/cloudflared-with-auto-dns-route:b471d0c
+    volumes:
+      - cloudflared-home:/root
+      - ./cloudflared/tunnel-config.yml:/image/tunnel-config.yml
+    networks:
+      - cloudflared-internet
```

これ、

```suggestion
    network_mode: "host"
```

としないとOnPネットワーク内のVM見えなさそう